lib/octokit/client/hooks.rb

```diff
@@ -88,7 +88,7 @@ def ping_hook(repo, hook_id, options = {})
 
       # Get single org hook
       #
-      # @param org [String] org parameter
+      # @param org [String] A GitHub organization
```

Comment:
Can we change this to state that it's the name or id of the organization? Much like we do repository?

Reply:
Yes! This was something I wanted to ask about. We actually use `A GitHub repository` for the repo parameter so this seemed consistent, but the repo parameter also accepts not just strings. I am all in favor of changing this though 👍